Mage/src/main/java/mage/game/GameState.java

```diff
@@ -96,6 +99,7 @@
     private Map<UUID, Integer> zoneChangeCounter = new HashMap<>();
     private Map<UUID, Card> copiedCards = new HashMap<>();
     private int permanentOrderNumber;
+    private Map<UUID, FilterCreaturePermanent> usePowerInsteadOfToughnessForDamageLethalityFilters = new HashMap<>();
```

Comment:
You must copy that values (copy each filter too). Search `copiedCards` for places with copy code.

Reply:
fixed, assuming you meant within the GameState's copy constructor and restore method

Reply:
also the clear method